examples/material_gallery/lib/gallery/app.dart

```diff
@@ -5,8 +5,45 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/scheduler.dart' show timeDilation;
 
+import '../demos.dart';
 import 'home.dart';
 
+final Map<String, WidgetBuilder> kRoutes = <String, WidgetBuilder>{
```

Comment:
The map you get out definitely has unique keys.  I'm not sure what happens if you put in non-unique keys.